__tests__/app.test.js

```diff
@@ -0,0 +1,51 @@
+const request = require('supertest')
+const app = require('../app.js')
+const db = require('../db/connection.js')
+const data = require('../db/data/test-data/index.js')
+const seed = require('../db/seeds/seed.js')
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('GET /api/topics', () => {
+    it('200: returns array of topic objects with correct keys', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) => {
+            expect(Object.keys(body.topics[0]).length).toBe(2)
+            expect(Object.keys(body.topics[0])).toContain('description', 'slug')
+        })
+    });
```

Comment:
These 200 tests are fine, but we could handle all of this with one test, if in the second 200 test you used a for each to check the appropriate properties were on each object